create_db.py

```diff
@@ -77,15 +76,16 @@ def insert_characters(session):
     logging.info("inserted %s characters", count)
 
 
-def insert_lines(session):
+def insert_lines(session, line_to_conv):
```

Comment:
`line_to_conv` could be renamed to `line_to_conv_mapping` to make it less ambiguous. `line_to_conv` could also be short for `line to convert`